_observing-your-data/event-analytics.md

```diff
@@ -39,21 +39,22 @@ To simplify query building, the **OpenSearch Assistant** toolkit offers an assis
 
 #### Enabling Query Assistant
 
-To enable **Query Assistant** in OpenSearch Dashboards, locate your copy of the `opensearch_dashboards.yml` file and set the following option:
+By default, **Query Assistant** is enabled in OpenSearch Dashboards. To configure Query Assistant, locate your copy of the `opensearch_dashboards.yml` file and set the following option:
 
-```
-observability.query_assist.enabled: true
```

Comment:
We can keep this removed part and say if they want to disable it, they can set this setting to false.

Reply:
I added a sentence about that on line 50.